zerver/openapi/zulip.yaml

```diff
@@ -24388,11 +24392,15 @@ paths:
                 name:
                   description: |
                     The new name of the channel folder.
+
+                    New name of channel folder can have maximum 60 characters.
```

Comment:
Rather than putting these lengths into the documentation, and making that the documented change, we should just reference a `max_channel_folder_name_length` field in the API response, like `max_stream_name_length`.